transactions/reset-epoch/2023/nov-9/README.md

```diff
@@ -0,0 +1,15 @@
+# Reset Epoch
+
+> Nov 9th, 2023
+
+Any time the network is sporked, the protocol state is manually initialized via the bootstrapping process.
+Therefore, we need to re-synchronize the FlowEpoch smart contract each time the network is sporked, using the [ResetEpoch](../../../../templates/reset_epoch_with_end_staking_auction.cdc) transaction.
```

Comment:
No need to use the end staking auction transaction here. just this one is fine: https://github.com/onflow/service-account/blob/main/templates/reset_epoch.cdc